src/main/java/org/killbill/billing/plugin/adyen/client/model/paymentinfo/KlarnaPaymentInfo.java

```diff
@@ -177,6 +190,27 @@ public String getAdditionalData() {
         return additionalData;
     }
 
+    private void hashIdentifiers() {
+        // hash sensitive identifiers
+        if (!identifierHashed) {
+            for (Seller seller : sellers) {
+                final String merchantId = seller.getMerchantId();
```

Comment:
need to check here if the seller is empty/null it will null pointer

Reply:
Not required. This form of for loop runs only of collection items are not empty.